design-library/source/_patterns/02-molecules/02-list-element/list-element.json

```diff
@@ -4,13 +4,23 @@
   "body": "The preemptive commitment leads to an agile feedback, while the game changer connects the dots to the end game. Our under-the-radar business platforms energize the market thinker.",
   "tag_list": {
     "1": {
-      "tag_name": "Cloud"
+      "tag_name": "Cloud",
+      "tag_icon_url": "/images/svg/politics-2.svg",
+      "tag_color": "#b10283",
```

Comment:
We should abstract these colors into a set of expected colors and names, e.g. "blue", "red", "light blue", "light green", and more, which have corresponding hex codes that complement the design

Reply:
Done